x-pack/plugins/siem/public/common/components/exceptions/viewer/exception_details.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexItem, EuiFlexGroup, EuiDescriptionList, EuiButtonEmpty } from '@elastic/eui';
+import React, { useMemo } from 'react';
+import styled, { css } from 'styled-components';
+import { transparentize } from 'polished';
+
+import { ExceptionListItemSchema } from '../types';
+import { getDescriptionListContent } from '../helpers';
+import * as i18n from '../translations';
+
+const StyledExceptionDetails = styled(EuiFlexItem)`
+  ${({ theme }) => css`
+    background-color: ${transparentize(0.95, theme.eui.euiColorPrimary)};
+    padding: 16px;
+
+    .euiDescriptionList__title.listTitle--width {
+      width: 40%;
+    }
+
+    .euiDescriptionList__description.listDescription--width {
+      width: 60%;
+    }
+  `}
+`;
+
+const ExceptionDetailsComponent = ({
+  showComments,
+  onCommentsClick,
+  exceptionItem,
+}: {
+  showComments: boolean;
+  exceptionItem: ExceptionListItemSchema;
+  onCommentsClick: () => void;
+}) => {
+  const descriptionList = useMemo(() => getDescriptionListContent(exceptionItem), [exceptionItem]);
+
+  const commentsSection = useMemo(() => {
+    const { comments } = exceptionItem;
+    if (comments !== null && comments && comments.length > 0) {
```

Comment:
nit: Most people use `!= null` which will get both the `null` and `undefined` use cases.

I rarely ever find myself or most people in general using the `=== null` or the `!== null` instead of the `== null` or the `!= null`

Also, either our types are incorrect or this is just extra null checking code because I see this for the types:

<img width="502" alt="Screen Shot 2020-06-03 at 10 41 05 AM" src="https://user-images.githubusercontent.com/1151048/83664116-cc4cb000-a586-11ea-9ae3-7b914dc632f3.png">

Where it states that it will always return back an array of elements. I would trust the types but if they are wrong then we fix the types and do boundary checking with decode better rather than adding extra null guards.

We do have boundary input and output checks with the request and response, I would trust them and remove the extra guards unless I'm missing something.

Reply:
Yea, sorry about that. This was leftover from when I was still pulling in the lists' exception schema that has `comment` instead of `comments`. Meant to clean up after adding the temporary exceptions schema.

Reply:
Thanks for pointing that out. Went through and updated instances of `!== null` to `!= null`.